slsDetectorSoftware/src/DetectorImpl.cpp

```diff
@@ -1309,13 +1309,26 @@ void DetectorImpl::startAcquisition(const bool blocking, Positions pos) {
         std::vector<int> masters;
         std::vector<int> slaves;
         getMasterSlaveList(pos, masters, slaves);
+        if (masters.empty()) {
+            throw RuntimeError("Cannot start acquisition in sync mode. No "
+                               "master module found");
+        }
         if (!slaves.empty()) {
             Parallel(&Module::startAcquisition, slaves);
         }
         if (!masters.empty()) {
```

Comment:
I guess the 
```
if (!masters.empty())
```
is now redundant since we throw if it it empty?

Reply:
done